main/openssh/APKBUILD

```diff
@@ -9,7 +9,7 @@ pkgdesc="Port of OpenBSD's free SSH release"
 url="http://www.openssh.org/portable.html"
 arch="all"
 license="as-is"
-options="suid"
+options="suid !check"
```

Comment:
If the provided tests are stupidly broken, then please add at least very basic check that would just run `sshd --version` or something like that, to at least verify that the binary exists and is not completely broken.

Also it’d be nice to add some brief detail into the commit message about how are the tests broken.